File: Prog1part3.c
#include <stdio.h>
#include <string.h>

int main(){
    int ms, sec, min, track;

    printf("Milliseconds: ");
    scanf("%d", &track);
    
    min = track / 60000;
    track = track - (60000 * min);

    sec = track / 1000;
    track = track - (1000 * sec);

    ms = track;

    printf("minutes(s): %d\nsecond(s): %d\nmilliseconds: %d\n", min, sec, ms);
    //printf("%d minute(s)\n%d second(s)\n%d milliseconds\n", min, sec, ms);

    return 0;
}

File: Prog1part3.java
import java.util.*;

public class Prog1part3{
    public static void main (String[] args){
        Scanner keyboard;
        keyboard = new Scanner(System.in);

        int ms, sec, min, track;

        System.out.print("Milliseconds: ");
        track = keyboard.nextInt();
        
        min = track / 60000;
        track = track - (60000 * min);

        sec = track / 1000;
        track = track - (1000 * sec);

        ms = track;

        System.out.println("Minute(s): " + min + "\nSecond(s): " + sec + "\nMilliseconds: " + ms);
    }
}
